graph_api/database_service.py

```diff
@@ -115,6 +117,49 @@ def get_nodes(self, label):
             label=label)
         return self.post_statement(get_statement)
 
+    def get_nodes_by_query(self, query: NodeRowsQueryIn):
+        """
+        Send to the database request to get nodes with given query
+        Args:
+            query (NodeRowsQueryIn): query to search by
+        Returns:
+            Result of request
+        """
+        return_list = []
+        where_list = []
+        match_list = []
+        if query.relations is not None:
+            for relation in query.relations:
+                statement = "(n_" + str(relation.begin_node_index) + ")-["
```

Comment:
Use f-strings, optionally `format()` to create new strings. 
https://realpython.com/python-f-strings/

Reply:
👌